opentok/archives.py

```diff
@@ -157,11 +157,5 @@ def attrs(self):
     def json(self):
         return json.dumps(self.attrs(), default=dthandler, indent=4)
 
-    def __getitem__(self, key):
```

Comment:
I didn't know what was it for and it is using a class that is not even imported in this module.  Are you sure that is needed?